designer/server/plugins/routes/api.ts

```diff
@@ -28,6 +29,10 @@ export const getFormWithId: ServerRoute = {
           formJson = values;
         }
       } catch (error) {
+        request.logger.error(
+          ["newConfig", "previewModeError"],
+          consoleMessages.publishError
+        );
```

Comment:
copypasta? This is just getting a form with id. not always to do with newConfig? There's also no confirmation if this is a "previewModeError". you would have to do error.statusCode === 404 (since this is the statusCode boom.notFound returns).